_pytest/_code/source.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import generators
+from __future__ import absolute_import, division, generators, print_function
```

Comment:
generators is not needed since python 2.3 - we may just want to remove it already ^^